nwb_conversion_tools/nwbconverter.py

```diff
@@ -60,25 +60,29 @@ def get_metadata(self):
             metadata = dict_deep_update(metadata, interface_metadata)
         return metadata
 
-    def run_conversion(self, metadata_dict, nwbfile_path=None, save_to_file=True, stub_test=False):
+    def run_conversion(self, metadata_dict, nwbfile_path=None, save_to_file=True, stub_test=False,
+                       **conversion_options):
         """Build nwbfile object, auto-populate with minimal values if missing."""
         nwbfile_kwargs = dict(
-                session_description="no description",
-                identifier=str(uuid.uuid4()),
-                session_start_time=datetime.now()
-            )
-
+            session_description="no description",
+            identifier=str(uuid.uuid4()),
+            session_start_time=datetime.now()
```

Comment:
Can we not write incorrect data please?

Reply:
session_description should not be filled if not informative, and session_start_time auto-fills the same way through pynwb. We still need the identifier here though, correct?

Reply:
session_description and session_start_time are both required arguments

Reply:
session_start_time does not autofill

Reply:
So the session_description should be filled in only by a specific sub-class of the NWBConverter, or otherwise modified by a user via the metadata fields? and session_start_time, identifier always being populated within this main class?

Reply:
Oh, sorry... I was confusing sess_start_time with that file_create_time field

Reply:
So it looks like this block of code is there to ensure the creation of an NWBFile, even if the metadata['NWBFile'] doesn't have the fields session_description or session_start_time, with the values listed here as the defaults. Would you instead just want it to throw an error if those fields are not present in the metadata?

Reply:
`session_description` can be safely filled with "no description". `session_start_time` is tricky though. It needs to be a `datetime`, and I don't think we can convert `pd.NaT` to ISO 8601. For that one I guess yeah we should just throw an error if it is missing.